docs/Manifest.toml

```diff
@@ -230,7 +236,7 @@ uuid = "c2caaa1d-32ae-4754-ba0d-80e7561362e9"
 version = "0.1.0"
 
 [[deps.ClimaCoreTempestRemap]]
-deps = ["ClimaComms", "ClimaCore", "Dates", "LinearAlgebra", "NCDatasets", "PkgVersion", "TempestRemap_jll", "Test"]
+deps = ["ClimaComms", "ClimaCommsMPI", "ClimaCore", "ClimaCorePlots", "Dates", "LinearAlgebra", "MPI", "NCDatasets", "PkgVersion", "Plots", "TempestRemap_jll", "Test"]
```

Comment:
It would be good for `ClimaCoreTempestRemap` to be independent of Plots.

Reply:
I think it needs Plots to generate the plots [here].(https://github.com/CliMA/ClimaCore.jl/pull/1107/files#diff-1e4f6dd4582cafce906c9edc5c99f075f342be492fd2ab82b278fb07a5efb0b1R125)

Reply:
Ahh yes, we should comment out these `plot` and `savefig` functions (since they are not actually being inspected by the CI, but they're still useful to have there for debugging).